### DISCUSSION TITLE
I've made myself a glow-in-the-dark, 3D printed cover!*

### DISCUSSION THREAD

**Discussion by ll-dasein-ll** [KBLiveSolutions/open.control#17] [General]
(*well, I got someone who knows what they're doing to make me one, anyway...)

And here it is :-)

![IMG_1279](https://user-images.githubusercontent.com/104435068/165828587-27b402e7-3931-4e83-aec7-8c17997fd9e3.jpg)

I used the .stl file off the Resources page (https://github.com/KBLiveSolutions/open.control/blob/main/ressources/OpenControlCover.stl) and asked the 3D printer technician at work if they could print it off.  After two attempts - the first one was printed with the infill at 20 which turned out to be too low - a second print was done with an infill of 50 (I think) and it's perfect.  I was worried the buttons were going to be a bit stiff but they're really sturdy, click the buttons on the board satisfyingly well and don't feel brittle at all.  I kind of expected them to feel like they might snap off at any moment.

![IMG_1280](https://user-images.githubusercontent.com/104435068/165832093-b45de254-b03f-47ae-9c92-7de991e14657.jpg)

The cover clicks neatly over the opencontrol board and it's a very snug fit.  Putting it on was a little nervewracking since I have no experience with how much "give" a 3D printed item when you're trying to click it over a circuit board, but it is surprisingly solid. Getting the caps off the encoders to slot the cover over them was the most difficult bit, until I found a pair of pliers to get some leverage - but then I accidentally pinged one of them off across the office and it took three of us about twenty mintes to find it again, lol.

![IMG_1278](https://user-images.githubusercontent.com/104435068/165828581-615b9548-e7c5-45fc-9051-21ba8cb23370.jpg)

The only slight issue with it after a few hours is that the printed cover seems to have become very, very slightly warped, so the board wobbles a bit on a flat surface - however, I noticed that the two back legs on the opencontrol have threads inside them, so I found a tiny screw with the same thread size and have screwed it in to raise up the side with the wobble so it's now perfectly stable again, and with adjustable height too!

I'd highly recommend doing this if you don't have a cover for your opencontrol board already - it makes the Pocket version feel much more sturdy and it looks great.  Mine also glows in the dark! (although I unfortunately can't get a good enough photo of it glowing here at home to bother sharing - I might try again tomorrow when I'm at my studio where there's basically no natural light).

![IMG_1276](https://user-images.githubusercontent.com/104435068/165828566-801d782d-d358-4e70-8d3f-5a01f85f973a.jpg)

**Comment by KBLiveSolutions** [comment]
Wow, awesome !
It looks as good as the original ! 👏

**Reply by ll-dasein-ll** [reply]
It's great!  I'm getting them to make a cover for my Axoloti board now, too :-)
